### PR TITLE
REV-32- finish dashboard page with shared tab, table, tooltip, accordion, dropdown menu

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
- figma link:
https://www.figma.com/file/22d3bKcFL0ipWIZJOUY3Km/reverifi?node-id=0%3A21013

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ services:
     volumes:
       - ${PWD}:/app
     ports:
-      - 80
+      - 3000:80
```
Comment (AhmadAlShwaiki) [inline docker-compose.yml:16]: Can we revert this?

File: src/components/dashboard/DashboardWrapper/index.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      </DashboardCardsContainer>
+
+      <DashboardHeading>Offers</DashboardHeading>
+      <OffersTable data={offersData} />
+
+      <TransactionsContainer>
+        <DashboardHeading>My Current Transactions</DashboardHeading>
+        <DropdownMenusContainer>
+          <DropdownMenu name="status" leftIcon={dropdownIcon1}>
+            <option value="">Status</option>
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/DashboardWrapper/index.jsx:65]: I think this should be a placeholder not an option. This fix can be done in the root component `DropdownMenu ` We can fix this in another PR or here.

```suggestion
```

File: src/components/dashboard/DashboardWrapper/index.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        ))}
+      </DashboardCardsContainer>
+
+      <DashboardHeading>Offers</DashboardHeading>
+      <OffersTable data={offersData} />
+
+      <TransactionsContainer>
+        <DashboardHeading>My Current Transactions</DashboardHeading>
+        <DropdownMenusContainer>
+          <DropdownMenu name="status" leftIcon={dropdownIcon1}>
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/DashboardWrapper/index.jsx:64]:
```suggestion
          <DropdownMenu name="Status" leftIcon={dropdownIcon1}>
```

File: src/components/dashboard/DashboardWrapper/index.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            <option value="">Status</option>
+            <option value="1">status 1</option>
+            <option value="1">status 3</option>
+          </DropdownMenu>
+          <DropdownMenu name="role" leftIcon={dropdownIcon2}>
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/DashboardWrapper/index.jsx:69]:
```suggestion
          <DropdownMenu name="Role" leftIcon={dropdownIcon2}>
```

File: src/components/dashboard/DashboardWrapper/index.jsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            <option value="">My Role</option>
+            <option value="1">seller</option>
+            <option value="2">buyer</option>
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/DashboardWrapper/index.jsx:72]:
```suggestion
            <option value="1">Seller</option>
            <option value="2">Buyer</option>
```

File: src/components/dashboard/DashboardWrapper/index.jsx (hunk 4 above)
Comment (AhmadAlShwaiki) [inline src/components/dashboard/DashboardWrapper/index.jsx:69]: Names are not clear here, please fix the above too if possible. 🙏 
```suggestion
          <DropdownMenu name="role" leftIcon={roleDropDownIcon}>
```

File: src/components/dashboard/OffersTable/index.jsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+          mortgageLetter,
+          myRole,
+          offerPrice,
+          submittedBy,
+          submittedOn,
+        }) => (
+          <TableRow key={title}>
+            <TableCell>
+              <Accordion title={title}>
+                <OfferDetails>
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/OffersTable/index.jsx:44]: I suppose we can replace these divs with `<ul>` or table <tr> ?
```suggestion
```

File: src/components/dashboard/OffersTable/offers-table.styles.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import { ReactComponent as counterIco } from 'assets/icons/dashboard-offers-counter.svg';
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const OfferDetails = styled.div`
+  background-color: ${colors.alabaster};
+  border-bottom: 0.0625rem solid ${colors.midGray};
```
Comment (AhmadAlShwaiki) [inline src/components/dashboard/OffersTable/offers-table.styles.js:8]:
```suggestion
  border-bottom: 0.06rem solid ${colors.midGray};
```

File: src/components/shared/Table/table-styles.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  text-align: left;
+`;
+
+export const TableCell = styled.td`
+  font-size: 0.8125rem;
+  padding: 1.6875rem 0.625rem;
+  text-align: ${({ iconsCell }) => iconsCell && 'right'};
+
+  ${mq.tablet`
+    padding: 1.6875rem;
```
Comment (AhmadAlShwaiki) [inline src/components/shared/Table/table-styles.js:36]:
```suggestion
    padding: 1.7rem;
```

File: src/components/shared/Tooltip/tooltip.styles.js [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    bottom: 100%;
+    left: 50%;
+    margin-left: -0.4375rem;
+  `,
+};
+
+export const TooltipText = styled.span`
+  background-color: ${colors.mineShaft};
+  border-radius: 0.625rem;
+  color: #fff;
```
Comment (AhmadAlShwaiki) [inline src/components/shared/Tooltip/tooltip.styles.js:37]:
```suggestion
  color: ${colors.white};
```

File: src/components/shared/Tooltip/tooltip.styles.js [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  z-index: 5;
+
+  bottom: ${({ bottom }) => bottom && `${bottom}rem`};
+  left: ${({ left }) => left && `${left}rem`};
+  right: ${({ right }) => right && `${right}rem`};
+  top: ${({ top }) => top && `${top}rem`};
+
+  &::after {
+    border-style: solid;
+    border-width: 0.4375rem;
```
Comment (AhmadAlShwaiki) [inline src/components/shared/Tooltip/tooltip.styles.js:52]:
```suggestion
    border-width: 0.5rem;
```

**Review by AhmadAlShwaiki — CHANGES_REQUESTED** [review]
Changes mainly focus on dropdown menu selection
css value auditing and standards
proper use of html elements

File: src/components/dashboard/DashboardWrapper/index.jsx (hunk 2 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/dashboard/DashboardWrapper/index.jsx:65]: This is a select tag it doesn't have placeholder

Comment (AhmadAlShwaiki) [reply src/components/dashboard/DashboardWrapper/index.jsx:65]: Please change them to a label, as these are not supposed to be an option

File: src/components/shared/DropdownMenu/index.jsx [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+        <StyledLabel htmlFor={name}>
+          {LabelIcon && LabelIcon}
+          <span>{label}</span>
+        </StyledLabel>
+      )}
+
+      <SelectContainer
+        leftIcon={leftIcon}
+        small={small}
+        smallBorderRadius={smallBorderRadius}
```
Comment (AhmadAlShwaiki) [inline src/components/shared/DropdownMenu/index.jsx:45]: Cant see why this is named this way 🤔 
```suggestion
        borderRadius={borderRadius}
```

Comment (IbrahimAldereni-socuimtech) [reply src/components/shared/DropdownMenu/index.jsx:45]: this is not a number, this is true/false value if the border raduis small or not

Comment (AhmadAlShwaiki) [reply src/components/shared/DropdownMenu/index.jsx:45]: Can't see why we would need this, please remove the prop and use the current style.

**Review by AhmadAlShwaiki — CHANGES_REQUESTED** [review]
Adjust DropdownMenu by removing border-radius styling prop

File: src/components/shared/DropdownMenu/index.jsx (hunk 11 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/shared/DropdownMenu/index.jsx:45]: this is a shared component sometimes the menu have small raduis and sometimes not

Comment (AhmadAlShwaiki) [reply src/components/shared/DropdownMenu/index.jsx:45]: can be adjusted via styled components. Just like the implementation of shared `<Button/>`

Comment (IbrahimAldereni-socuimtech) [reply src/components/shared/DropdownMenu/index.jsx:45]: yes i already done that its adjust it by styled component ??

**Pull request merged**